libraries/core-react/src/Button/Button.jsx

```diff
@@ -7,6 +7,14 @@ import { typographyTemplate } from '../_common/templates'
 
 const { colors } = button
 
+// display:grid; does not work on Webkit browser engine, so we have to wrap content in element where css-grid works
+const WebkitFix = styled.span`
```

Comment:
Would call it something more generic – `ButtonInner` for example. Apart from that LGTM 👍